test/integration/testStatement.js

```diff
@@ -205,7 +205,7 @@ describe('Statement Tests', function ()
 
                 assert.strictEqual(statement.getNumRows(), 1);
                 assert.ok(statement.getSessionState());
-                assert.ok(statement.getStatementId());
+                assert.ok(statement.getQueryId());
```

Comment:
shouldn't we keep both assertions for statementId and queryId? and even check that they are equal as a next assertion?

Reply:
added testing for both getStatementId and getQueryId